addons/purchase_requisition/tests/test_cancel_purchase_requisition.py

```diff
@@ -0,0 +1,21 @@
+from odoo.addons.purchase_requisition.tests.common import TestPurchaseRequisitionCommon
+
+
+class TestCancelPurchaseRequisition(TestPurchaseRequisitionCommon):
+
+    def setUp(self):
+        super(TestCancelPurchaseRequisition, self).setUp()
+        self.product_09 = self.env.ref('product.product_product_9')
+        self.product_09_uom = self.env.ref('product.product_uom_unit')
```

Comment:
In test case can not we use self.ref where we have to use id, self.env.ref give recordset. Check in all files for same